data/json/items/containers/containers.json

```diff
@@ -1271,6 +1271,33 @@
     "symbol": ")",
     "color": "brown"
   },
+  {
+    "id": "box_oversize_reinforced",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "reinforced oversize cardboard box", "str_pl": "reinforced oversize cardboard boxes" },
+    "looks_like": "box_small",
+    "description": "An oversize cardboard box, doubled up with a second layer of cardboard and lots of duct tape.  It can't be folded anymore.",
+    "ascii_picture": "box_oversize",
+    "weight": "4 kg",
+    "volume": "205000 ml",
+    "longest_side": "201 cm",
+    "//": "(200x50x20)",
+    "price": "0 cent",
```

Comment:
I would suggest changing the pre-apoc price to somewhere around $10. I couldn't find any boxes this large online, but smaller ones seem to cost at least $10 [source](https://www.uline.com/BL_412/Bulk-Cargo-Containers).
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/151102088/49a4afdc-a1a7-4717-96e3-898ae7a72b67)

Reply:
I did not change the pre-apocalypse price initially because this box is not an existing item, it's two boxes duct taped together by a survivor, and as such isn't really something that was for sale previously.  But since this doesn't matter, I changed the price as you suggested to $10.